db/changes.go

```diff
@@ -775,7 +775,7 @@ func (db *Database) SimpleMultiChangesFeed(chans base.Set, options ChangesOption
 
 			if options.Revocations && db.user != nil {
 				revocationCheckSeq := options.Since.SafeSequence()
-				if options.Since.TriggeredBy > 0 {
+				if options.Since.TriggeredBy > 0 && revocationCheckSeq > options.Since.TriggeredBy-1 {
```

Comment:
I think this condition merits a comment to provide some context on what the second condition is doing.